onnx/test/test_external_data.py

```diff
@@ -285,7 +285,7 @@ def create_test_model_both(self):  # type: () -> Tuple[Text, ModelProto]
     # 2GB models would fail onnx.checker with ModelProto but pass with model path
     def test_check_model_by_model(self):  # type: () -> None
         model = onnx.load_model(self.model_filename, load_external_data=False)
-        with pytest.raises(ValueError):
+        with pytest.raises((ValueError, MemoryError)):
```

Comment:
For memory error add a log statement stating --> because of mem limitation this test was not executed.

Reply:
Good idea. Just added.